_posts/2016-11-01-devfest-cfp-cfv.markdown

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title:  "Call for Papers and Volunteers"
+date:   2016-09-01 23:59:59
+isStaticPost: false
+---
+
+We hereby open our this years Call for Particpants!
```

Comment:
Doesn't exactly roll off the tongue. Maybe "We hereby open the Call for Participation for the 2016 edition of DevFest Berlin"?